cl/sentinel/service/service.go

```diff
@@ -206,15 +211,21 @@ func (s *SentinelServer) requestPeer(ctx context.Context, pid peer.ID, req *sent
 	}
 	// known error codes, just remove the peer
 	if isError == 3 || isError == 2 {
```

Comment:
sorry again curious the response code 2,3 here. Where is the definition of these numbers?

Reply:
https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#responding-side

```
The response code can have one of the following values, encoded as a single unsigned byte:

0: Success -- a normal response follows, with contents matching the expected message schema and encoding specified in the request.
1: InvalidRequest -- the contents of the request are semantically invalid, or the payload is malformed, or could not be understood. The response payload adheres to the ErrorMessage schema (described below).
2: ServerError -- the responder encountered an error while processing the request. The response payload adheres to the ErrorMessage schema (described below).
3: ResourceUnavailable -- the responder does not have requested resource. The response payload adheres to the ErrorMessage schema (described below). Note: This response code is only valid as a response where specified.
```